src/Apps/Legion/Components/DataPusher/DataPusher.types.ts

```diff
@@ -102,3 +103,14 @@ export interface IReactSelectOption {
     label: string;
     __isNew__?: boolean;
 }
+
+// Temporary types - will clear when data pusher is updated to new data patterns
+export interface IModelExtended extends IModel {
```

Comment:
do we still need these types now that we migrated?

Reply:
DataPusher still has those types, we need to remove them in another PR.